src/viewProviders/resources.ts

```diff
@@ -58,7 +63,11 @@ import { filterItems, itemMatchesSearch, SEARCH_DECORATION_URI_SCHEME } from "./
 
 const logger = new Logger("viewProviders.resources");
 
-type CCloudResources = CCloudEnvironment | CCloudKafkaCluster | CCloudSchemaRegistry;
+type CCloudResources =
+  | CCloudEnvironment
+  | CCloudKafkaCluster
+  | CCloudSchemaRegistry
+  | CCloudFlinkComputePool;
 type LocalResources = LocalEnvironment | LocalKafkaCluster | LocalSchemaRegistry;
 type DirectResources = DirectEnvironment | DirectKafkaCluster | DirectSchemaRegistry;
```

Comment:
Cleaning up this section and the types used inside `getChildren()` is out of scope for this branch and should be tackled separately, but I think we're getting closer.